internal/command/container.go

```diff
@@ -0,0 +1,393 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fyne-io/fyne-cross/internal/icon"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+	"golang.org/x/sys/execabs"
+)
+
+const (
+	// fyneBin is the path of the fyne binary into the docker image
+	fyneBin = "/usr/local/bin/fyne"
+	// gowindresBin is the path of the gowindres binary into the docker image
+	gowindresBin = "/usr/local/bin/gowindres"
+)
+
+type ContainerRunner interface {
+	NewImageContainer(arch Architecture, OS string, image string) ContainerImage
```

Comment:
Not sure about having a constructor defined as part of an interface

Reply:
Agreed, that isn't the Go pattern, It is not clear what `ContainerRunner` is describing.

Reply:
Thinking about it, `ContainerRunner` is basically the engine definition (podman, docker, kubernetes).

Reply:
Ah, so `Engine` could work, remove the duplication that's tripping me up?
Could use `Create` instead of `New` so it's not looking like a constructor?
Also I realise here there is `ImageContainer` vs `ContainerImage` - which is desired?

Reply:
I think `ContainerImage`.